sale_fully_invoiced/tests/test_sale_fully_invoiced.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2022 ForgeFlow, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo.tests import common, tagged
+
+
+@tagged("-at_install", "post_install")
+class TestSaleFullyDelivered(common.TransactionCase):
+    def setUp(self):
```

Comment:
@AaronHForgeFlow Could you change this to classmethod ?

Reply:
@rousseldenis did that :) sorry for the delay